exercises/practice/kindergarten-garden/.docs/instructions.append.md

```diff
@@ -1,4 +1,5 @@
 # Instructions append
 
-In this exercise you're going to be using a concept called dynamic programming. https://en.wikipedia.org/wiki/Dynamic_programming
-Your solution could use the class method `#define_singleton_method` or the combination of `#method_missing` and `#respond_to_missing`.
+The exercise requires you to write a class that provides a single method for each child in the kindergarten, respectively.
+You can make your work easier and less repetitive if, instead of manually defining 12 different methods, you use a bit of metaprogramming.
```

Comment:
```suggestion
  You can make your work less repetitive (and more fun!) if, instead of manually defining 12 different methods, you use a bit of metaprogramming.
```

Reply:
I have no problems with this and the above suggestions. Are you, by the way, OK with amending the commit even for such more substantial changes, or do you prefer a dedicated new commit?

Reply:
It's probably better to have a different PR for bigger changes. These appends should be small hints really, not explanations about how to do things though, so I wouldn't expect the text to be too long here :)